src/pages/Pages.css

```diff
@@ -48,3 +48,10 @@
   font-size: 14px;
   font-weight: 400;
 }
+.nav-button {
```

Comment:
This issue keeps on coming back. There might be something wrong on how you are creating your branch? If you look at the file it became `Pages.css` again (capital P).

<img width="1262" alt="Compiled with problems" src="https://github.com/daniel-august-mentorship/Demo-School/assets/6497134/4b180904-dcc0-486d-9b16-f765022a7755">

Reply:
Sorry Daniel. but I don't know why this issue occurred. In my file it appears as a lowercase p.
Can you advise why this issue occured.

![pages](https://github.com/daniel-august-mentorship/Demo-School/assets/115160155/4e8cbd31-6bc7-4ff8-bb99-503f84a36e3f)

Reply:
I found out why this issue occurred.
I will change lowercase p. thank you!

![pagesissue](https://github.com/daniel-august-mentorship/Demo-School/assets/115160155/413ec693-34df-46f8-8c65-9196acbdb3a3)